pokeAPI.js

```diff
@@ -0,0 +1,71 @@
+try {
+    const selectPoke = async (url, input) => { 
+        let randomPokeArray = []
```

Comment:
remember to have these in try blocks so you can be able to have catch in case of errors